File: CleverFerretV2/feature/ai/src/main/java/com/cleverferret/v2/feature/ai/audit/AiAuditEvent.java
package com.cleverferret.v2.feature.ai.audit;

import com.cleverferret.v2.feature.ai.model.AiCapability;

import java.time.Instant;

public record AiAuditEvent(
        Instant timestamp,
        String requester,
        String provider,
        AiCapability capability,
        boolean outbound,
        String status
) {}

File: CleverFerretV2/feature/ai/src/main/java/com/cleverferret/v2/feature/ai/audit/AiAuditEvent.kt
package com.cleverferret.v2.feature.ai.audit

import com.cleverferret.v2.feature.ai.model.AiCapability
import java.time.Instant

data class AiAuditEvent(
    val timestamp: Instant,
    val requester: String,
    val provider: String,
    val capability: AiCapability,
    val outbound: Boolean,
    val status: String
)
